src/imp/tracer_imp.js

```diff
@@ -183,24 +185,57 @@ export default class TracerImp extends EventEmitter {
     }
 
     startSpan(fields) {
-        let spanImp = new SpanImp(this);
+        // First, assemble the SpanContextImp for our SpanImp.
+        //
+        // Normalize `fields.reference` and `fields.references` (at most one
+        // will be defined).
+        let refs = fields.references;
+        if (fields.reference) {
+            refs = [fields.reference];
+        }
+        let parCtxImp = null;
+        if (refs) {
+            for (let i = 0; i < refs.length; i++) {
+                let ref = refs[i];
+                if (ref.type() === OpenTracing.REFERENCE_CHILD_OF ||
+                        ref.type() === OpenTracing.REFERENCE_FOLLOWS_FROM) {
+                    parCtxImp = ref.spanContext().imp();
+                    break;
```

Comment:
note that we still only support a single parent (across languages), even though OpenTracing theoretically supports true DAG traces now.